.github/workflows/images.yml

```diff
@@ -4,15 +4,19 @@ on:
   workflow_dispatch:
     inputs:
       implementation:
+        description: The name of an implementation.
+        required: false
+        type: string
+      version:
         description: >
-          The name of an implementation.
+          A specific version of the implementation you want to build.
```

Comment:
Is `implementation` required when providing this `version` option (i.e. it's only optional if this isn't provided)?

Reply:
I've made it so that either the user provides just the implementation / implementation + version / nothing at all. Each of these scenarios are handled in the if then logic.

Reply:
If the user just provides the version without specifying any implementation then should I throw some error and exit ?

Reply:
Probably yes, though I would slightly doubt GH Actions can represent that constraint so yeah you'll have to simply do it in the shell code.

Reply:
Done @Julian.